packages/@aws-cdk/aws-amplify/README.md

```diff
@@ -38,7 +38,7 @@ const amplifyApp = new amplify.App(this, 'MyApp', {
     repository: '<repo>',
     oauthToken: cdk.SecretValue.secretsManager('my-github-token')
   }),
-  buildSpec: codebuild.BuildSpec.fromObject({ // Alternatively add a `amplify.yml` to the repo
+  buildSpec: codebuild.BuildSpec.yamlFromObject({ // Alternatively add a `amplify.yml` to the repo
```

Comment:
We also need to update the ReadMe of the CodeBuild module.

Reply:
I've updated the readme, but here I have some doubts

I'm not familiar with codebuild, so they might be a bit stupid :-) .

---

Is `buildSpec` from codebuild expected to be a yaml?

The old json function goes over until the function [tokenAwareStringify](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/core/lib/private/cloudformation-lang.ts#L142). 

Is there something that codebuild would expect from this token function that the yaml would not cover yet? Not sure if it is more general for cloudformation or if codebuild needs something from it.

```
Stack.of(ctx.scope).toJsonString(this.spec, 2),
> CloudFormationLang.toJSON(obj, space).toString();
>> tokenAwareStringify()
```